src/town/a3.py

```diff
@@ -44,15 +45,15 @@ def stash_is_open_func():
             found = self._template_finder.search("INVENTORY_GOLD_BTN", img, roi=self._config.ui_roi["gold_btn"]).valid
             found |= self._template_finder.search("INVENTORY_GOLD_BTN", img, roi=self._config.ui_roi["gold_btn_stash"]).valid
             return found
-        if not self._char.select_by_template("A3_STASH", stash_is_open_func, telekinesis=True):
```

Comment:
It defaults to true

Reply:
it's that way in the a5.py. i mostly copied it from there.